[AssetMapper] Automatically preload CSS files if WebLink available (#19021)
symfony/symfony-docs: frontend/asset_mapper.rst

    live in your ``importmap.php`` file, so their preload setting is handled
    explicitly in that file.

For used CSS files, a "preload" tag will automatically be rendered onto your page
when the :doc:`WebLink Component </web_link>` is available.

.. versionadded:: 6.4

>     The feature to automatically preload CSS files when WebLink is available was

Reviewer: ```suggestion
    Automatic preloading of CSS files when WebLink is available was
```
